flink/src/test/java/org/apache/iceberg/flink/data/RandomData.java

```diff
@@ -88,20 +104,153 @@ public Row next() {
     };
   }
 
+  private static Iterable<RowData> generateRowData(Schema schema, int numRecords,
```

Comment:
We could use `RandomRowData#generate`  when rebasing the patch https://github.com/apache/iceberg/pull/1320/files#diff-4b2a9fd76495497db9212d74bf03f671R33.

Reply:
Will do.